### DISCUSSION TITLE
Can pinned tabs display as a list, rather than as a grid?

### DISCUSSION THREAD

**Discussion by roke-julian-lockhart** [mbnuqw/sidebery#1874] [Q&A] [answered]
Currently, they display as a grid:

![image](https://github.com/user-attachments/assets/17bb6858-0484-4abe-b321-7b7541449706)

**Comment by albino1** [comment]
Under Settings -> Tabs -> Pinned Tabs try setting `Pinned tabs position` to `globally - top` and set `Show titles of pinned tabs` to `on`.

**Reply by roke-julian-lockhart** [reply, marked as answer]
> "Under Settings" -> "Tabs" -> "Pinned Tabs", set "Show titles of pinned tabs" to "on".

Thanks, @albino1 — “Show titles on pinned tabs” provides this capability.
